src/app/features/notifications/notification-item/notification-item.component.html

```diff
@@ -0,0 +1,36 @@
+<div
+  class="flex-row justify-space-between align-center notification-item-wrapper"
+  [ngClass]="{ 'read-notification': notification.readStatus }"
+>
+  <div class="notification-details">
+    <strong i18n class="notification-item">{{ notification.title }}</strong>
+    <p i18n class="notification-item">{{ notification.body }}</p>
+    <p i18n class="notification-time">{{ notification.created.at }}</p>
+  </div>
+
+  <!-- Blue indicator for unread notifications -->
+  <span
+    *ngIf="!notification.readStatus"
+    class="unread-indicator"
+    matTooltip="Unread notification"
+    i18n-matTooltip="Unread notification"
+    matBadgeColor="accent"
+    [matMenuTriggerFor]="notificationMenu"
+    (click)="$event.stopPropagation()"
```

Comment:
When I remove ` $event.stopPropagation()`, clicking on any menu item causes the menu to close automatically. For example, when I click the blue dot, the new menu item list opens, but the previous one closes. In some cases, however, I don't want the menu item to close automatically. That's why I added `$event.stopPropagation()` to prevent the menu item from closing on its own.

Reply:
Ah, I see ... Seems a bit tedious to put it on almost every event handler but if we need it, that's okay. (might be any way to stop this more centrally without all the repeated  stopPropagation calls?)